src/features/movies/MovieList/Container/index.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
```

Comment:
Skoro '<Wrapper>'  obejmuje całą zawartość tej podstrony, zrób z niego znacznik '<main>'.

Reply:
Faktycznie, tak powinno być, coś mi musiało umknąć, gdy to poprawiałem